src/ProjectSystemTools/BuildLogging/BuildLoggingToolWindow.cs

```diff
@@ -192,7 +195,7 @@ private void SaveLogs()
             {
                 return;
             }
-            ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+            ThreadHelper.JoinableTaskFactory.Run(async () =>
```

Comment:
Some of these conversions from `RunAsync` to `Run` might not be necessary. In this case, for example, the caller doesn't really need to wait for completion.